app/components/Registration/AccountRegistrationConfirm.jsx

```diff
@@ -0,0 +1,217 @@
+import React from "react";
+import PropTypes from "prop-types";
+import {connect} from "alt-react";
+import Clipboard from "react-clipboard.js";
+import AccountActions from "actions/AccountActions";
+import AccountStore from "stores/AccountStore";
+import WalletDb from "stores/WalletDb";
+import notify from "actions/NotificationActions";
+import TransactionConfirmStore from "stores/TransactionConfirmStore";
+import Translate from "react-translate-component";
+import {FetchChain} from "bitsharesjs/es";
+import WalletUnlockActions from "actions/WalletUnlockActions";
+import Icon from "components/Icon/Icon";
+import ls from "common/localStorage";
+
+const STORAGE_KEY = "__graphene__";
```

Comment:
Please work with Storages only inside Stores by using Actions.

Reply:
Thanks, good catch, actually this code is excess, as it used for Openledger's Log out. Fixed.